### PR TITLE
Add IsIPAddress Rule issue#10

### PR DESCRIPTION
**Pull request by edcapetti-zz**
Created IsIPAddress rule and test.

### CODE REVIEW

File: src/Rule/IsIPAddress.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+use Morebec\Validator\ValidationRuleInterface;
+
+/**
+ * Validates that a value is an Ip Address
+ */
+class IsIPAddress implements ValidationRuleInterface
+{
+    /**
+     * @var string
```
Comment (jwillp) [inline src/Rule/IsIPAddress.php]: Can you set the docstring to `string|null` please?

File: src/Rule/IsIPAddress.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    {
+        $this->message = $message;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function validate($v): bool
+    {
+        return filter_var($v, FILTER_VALIDATE_EMAIL) !== false;
```
Comment (jwillp) [inline src/Rule/IsIPAddress.php]: The filter here should be `FILTER_VALIDATE_IP`.

**Review by jwillp — CHANGES_REQUESTED** [review]
Hi @edcaracas, thank you for this second contribution!
See my comments.

File: src/Rule/IsIPAddress.php (hunk 1 above)
Comment (edcapetti-zz) [reply src/Rule/IsIPAddress.php]: Fixed docstring
https://github.com/Morebec/Validator/pull/14/commits/90204edf801163c8c75acda36ef702be6c49e705

**Review by jwillp — APPROVED** [review]

**Comment by edcapetti-zz** [comment]
Thank you.

**Comment by jwillp** [comment]
Thank **you** :)

**Pull request merged**